tests/unit/10_build-env.bats

```diff
@@ -44,6 +44,20 @@ teardown() {
   assert_success
 }
 
+@test "(build-env) buildpack if Dockerfile present" {
```

Comment:
@michaelshobbs: My way of thinking here was, that we know that the `Dockerfile` application is NodeJS, so if I specify the PHP buildpack it should fail. 

There might be a better solution to this, so if you have anything in your mind, please let me know.